src/core/migrator.js

```diff
@@ -61,6 +61,24 @@ export function getMigrator (type, args) {
 
     return sequelize
       .authenticate()
+      .then(() => {
+        // Check if this is a PostgreSQL run and if there is a custom schema specified, and if there is, check if it's
+        // been created. If not, attempt to create it.
+        if (helpers.version.getDialectName() === 'pg') {
+          const customSchemaName = helpers.umzug.getSchema('migration');
+          if (customSchemaName && customSchemaName !== 'public') {
+            return sequelize.showAllSchemas().then(schemas => {
```

Comment:
Use `createSchema` directly which handles existing schema with `IF NOT EXISTS`  https://github.com/sequelize/sequelize/blob/aa92764f953eeefbf194a6e2b765552adfa73bda/lib/query-interface.js#L35